### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/feature/page-info.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#816] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/feature/page-info.html

vuepress-theme-hope 实现了文章信息展示。

https://theme-hope.vuejs.press/zh/guide/feature/page-info.html

**Comment by 1536316364** [comment]
123123

**Comment by caofangqi** [comment]
求教 为什么没有显示字数

**Reply by CuteReimu** [reply]
默认是不显示字数的，你可以在全局主题选项中修改 `pageInfo`，将 `Word` 加入，或者修改页面的 frontmatter ，以达到显示字数的目的。

**Comment by QingFeng-awa** [comment]
写作日期目前没有办法禁用(让它不显示)吗

**Reply by itwanger** [reply]
同问呢

**Reply by CuteReimu** [reply]
修改全局主题选项中的 `pageInfo` 即可，默认值是包含 `Date` 的。手动指定一下，不包含 `Date` 即可，例如：`pageInfo: ["Author", "ReadingTime"]`

**Comment by YuleBest** [comment]
可以自定义 Tags 的颜色吗

**Reply by CuteReimu** [reply]
单独每一个的应该是不能。
但是，你要是想换一种颜色，应该是能直接换吧？我不确定，回头看看。
主题没有考虑过专门为某个分类或者某个标签设置特殊颜色的规则。
